packages/embed/src/js/smartselfie-auth.js

```diff
@@ -248,6 +249,7 @@ import { version as sdkVersion } from '../../package.json';
       callback_url: config.callback_url,
       token: config.token,
       partner_params,
+      ...getMetadata(),
```

Comment:
We will need to have a metadata key `metadata: getMetadata(),` if this is already taken care of then that's fine